### PR TITLE
add Natural Earth to registry

### PR DESCRIPTION
**Pull request by nvkelso**
stub of natural earth registry

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: f66e35d1ba4139e16ea6cb947f0175b34557b6fd
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e81601df-a076-4c5d-b583-2411362e42c9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 5b14ae3a921f8a6e2daa805cb18b9e1d0adf90c8
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=c448321f-3591-4359-bd14-eafecce7d7c3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 244e4e455ba17d85668f5947a17e1c53b0551c76
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=a19f5a8d-8865-461f-9ef6-628e90ba416f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 17aafbc3a261183392020032b437c4195b4f4007
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=0a0ee3a1-5879-4508-a34b-806fd9d5f72b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: c2c3115ba16a6c8d6242c3082164d3e7cede1530
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=05e674c8-843d-4635-ac50-82778bcd1bc3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 395c7bb30ab35d714589f4e308bc743b63509dd2
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=ab0dc1bf-8c67-4bb6-8b3f-f9db6ef644d0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 71e4431b699fd2cfc30e7e369b9116fa67de94c3
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=9d6b5806-13f7-40b4-a227-f8ec70f768fa%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 397cf28341e76e6e76c6b51ee6d248877ad6192f
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=4e1b6676-c042-4818-805c-46d246870154%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 51921300542bbb6b1fb753508dd8e252e6b6f6e1
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=3e86f1df-f0f2-45c2-bb1f-81fe0d876fa3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 672bca3016aa5fa6d2a769bf7d5ccc70aff767d0
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=118df84c-5ac6-437c-980e-8fa043f8487d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 40e13d233083c8bd4e9d8bb54a544409aad35228
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=97fc8070-6495-475c-9d9c-4a0dad79ae54%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 169e49457c9ab210290b2e08f16c583fa102973c
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=b850d33c-5e5c-4198-af42-6e9e6bc7e284%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: d7766b2ab2f16300b6c1ea76c561a960d0295e41
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=88564ede-a78c-4bb0-be9a-b4b752b96155%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/naturalearth.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - earth observation
+  - population
+License: Public Domain. https://www.naturalearthdata.com/about/terms-of-use/
+Resources:
+  - Description: Shapefile downloads in ZIP archives
+    ARN: arn:aws:s3:::naturalearth
+    Region: us-west-2
+    Type: S3 Bucket
+    RequesterPays: False
+    ControlledAccess: False
```
Comment (zflamig) [inline datasets/naturalearth.yaml:22]: You can just delete this one, it expects a string here for information on how to access the data if its controlled access

Comment (nvkelso) [reply datasets/naturalearth.yaml]: Fixed via https://github.com/awslabs/open-data-registry/pull/853/commits/2a81509b8fb068253a75dcf9afc4d888048e831a

**Comment by nvkelso** [comment]
There's obviously been some churn in the YAML file here, but it seems to pass now.

If there's anything else you want me to add let me know.

**Comment by barronh** [comment]
+1 
Having this as a part of the registry would be useful for many public users.

**Comment by jflasher** [comment]
Thanks @nvkelso and no worries on the commits, this is what squash & merge is for. :)

**Pull request merged**